lib/api/list_implementation.dart

```diff
@@ -32,60 +34,55 @@ class ListAPIService extends APIService implements ListService {
       if (response == null) return null;
       final map = response.body;
       if (map.containsKey('id')) {
-        return client
-            .get("/lists/$listId/tasks")
-            .then((tasks) {
-              map['tasks'] = tasks?.body;
-              return TaskList.fromJson(map);
-            });
+        return client.get("/lists/$listId/tasks").then((tasks) {
+          map['tasks'] = tasks?.body;
+          return TaskList.fromJson(map);
+        });
       }
       return TaskList.fromJson(map);
     });
   }
 
   @override
   Future<List<TaskList>?> getAll() {
-    return client.get('/lists').then(
-        (list) {
-          if (list == null || list.statusCode != 200) return null;
-          if (list.body.toString().isEmpty)
-            return Future.value([]);
-          print(list.statusCode);
-          return convertList(list.body, (result) => TaskList.fromJson(result));});
+    return client.get('/lists').then((list) {
+      if (list == null || list.statusCode != 200) return null;
+      if (list.body.toString().isEmpty) return Future.value([]);
+      print(list.statusCode);
+      return convertList(list.body, (result) => TaskList.fromJson(result));
+    });
   }
 
   @override
   Future<List<TaskList>?> getByNamespace(int namespaceId) {
```

Comment:
@Benimautner Is this still used? The api endpoints don't work anymore.

Reply:
No, there's a bunch of dead code for editing lists & namespaces which should be cleaned. It's still in there bc I wanted to check whether I could adapt the pages to edit projects.